Source/Lib/Codec/EbInitialRateControlProcess.c

```diff
@@ -370,42 +370,6 @@ EB_ERRORTYPE InitialRateControlContextCtor(
 	return EB_ErrorNone;
 }
 
-/************************************************
-* Release Pa Reference Objects
-** Check if reference pictures are needed
-** release them when appropriate
-************************************************/
-void EbHevcReleasePaReferenceObjects(
-	PictureParentControlSet_t         *pictureControlSetPtr)
-{
-	// PA Reference Pictures
-	EB_U32                             numOfListToSearch;
-	EB_U32                             listIndex;
-	if (pictureControlSetPtr->sliceType != EB_I_PICTURE) {
-
-		numOfListToSearch = (pictureControlSetPtr->sliceType == EB_P_PICTURE) ? REF_LIST_0 : REF_LIST_1;
-
-		// List Loop
-		for (listIndex = REF_LIST_0; listIndex <= numOfListToSearch; ++listIndex) {
-
-				// Release PA Reference Pictures
-				if (pictureControlSetPtr->refPaPicPtrArray[listIndex] != EB_NULL) {
-
-                    EbReleaseObject(((EbPaReferenceObject_t*)pictureControlSetPtr->refPaPicPtrArray[listIndex]->objectPtr)->pPcsPtr->pPcsWrapperPtr);
-					EbReleaseObject(pictureControlSetPtr->refPaPicPtrArray[listIndex]);
-				}
-		}
-	}
```

Comment:
Moved this to EncDec kernel (Line 3211 ~ 3223), where the picture(s) referencing to the current PPCS have completed encoding (not used in latter EntropyCoding and Packetization any more).